src/include/cache.i

```diff
@@ -133,93 +133,6 @@ __wt_cache_status(WT_SESSION_IMPL *session, int *evictp, int *dirtyp)
 }
 
 /*
- * __wt_eviction_check --
- *	Wake the eviction server if necessary.
- */
-static inline int
-__wt_eviction_check(WT_SESSION_IMPL *session, int *fullp, int wake)
```

Comment:
Previously, the fast checks (when the cache isn't full) were inlined.  What's the advantage to paying for a function call?